cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js

```diff
@@ -1,7 +1,7 @@
 import ExpandableTransition from '@cfpb/cfpb-expandables/src/ExpandableTransition';
 
 const MOBILE_COLLAPSED_CLASS = 'o-expandable__mobile-collapsed';
-const MOBILE_WIDTH = 800;
+const MOBILE_WIDTH = 900;
```

Comment:
This caused some unit tests to fail. These numbers (800 and 801) may need to be updated to 900 and 901: https://github.com/cfpb/consumerfinance.gov/blob/main/test/unit_tests/apps/teachers-digital-platform/js/expandable-mobile-spec.js#L92-L112